ui/admin/app/components/form/field/list-wrapper/textarea/index.hbs

```diff
@@ -0,0 +1,63 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{#let (unique-id) as |valueHeaderID|}}
+  <Hds::Table class='list-wrapper-field' name={{@name}}>
+
+    <:body as |B|>
+
+      {{#each @options as |option index|}}
+        <B.Tr>
+          <B.Td>
+            <Hds::Form::Textarea::Field
+              @value={{option.value}}
+              @type='text'
+              disabled={{@disabled}}
+              aria-label={{valueHeaderID}}
```

Comment:
Same here, don't use an ID for the aria-label

Reply:
Agree with this. I will make the changes